docs/memory/sgx-epc.md

```diff
@@ -0,0 +1,22 @@
+# SGX EPC Limit Plugin
+
+The sgx-epc NRI plugin allows control over SGX encrypted page cache usage
+using the cgroup v2 misc controller and pod annotations.
+
+## Annotations
+
+You can annotate encrypted page cache limit for every container in the pod,
+or just a specific container using the following annotation notations:
+
+```yaml
+...
+metadata:
+  annotations:
+    # for all containers in the pod
+    epc-limit.nri.io/pod: "32768"
+    # alternative notation for all containers in the pod
+    epc-limit.nri.io: "8192"
```

Comment:
users are expected to set per-container `sgx.intel.com/epc` limits which we'd take to `epc-limit.nri.io/container.c0` annotations. Are these about pod-level `misc` setting (e.g., a sum of all per-container requests within the pod)?

Reply:
No, the '/pod' (and its alternative) notation applies to any container in the pod. It's simply a shorthand notation that can be used instead of adding for every container CTR in the pod a container-specific `.../container.$CTR: $limit`.

Reply:
OK, hmm. There's probably no use for that functionality.

I could think of a case where we'd allow a policy that takes the sum of epc resource requests and adds that to a sandbox level shared resource...